src/path.cpp

```diff
@@ -165,23 +167,13 @@ namespace {
 
 	native_path_string convert_to_native_path_string(std::string const& path)
 	{
-#ifdef TORRENT_WINDOWS
 #if TORRENT_USE_UNC_PATHS
-		std::string prepared_path;
 		// UNC paths must be absolute
 		// network paths are already UNC paths
-		if (path.substr(0,2) == "\\\\")
-			prepared_path = path;
-		else
-		{
-			std::string sep_path = path;
-			std::replace(sep_path.begin(), sep_path.end(), '/', '\\');
-			prepared_path = "\\\\?\\" + (is_complete(sep_path) ? sep_path : complete(sep_path));
-		}
-#else
-		std::string prepared_path = path;
+		std::string prepared_path = complete(path);
+		if (prepared_path.substr(0,2) != "\\\\")
+			prepared_path = "\\\\?\\" + prepared_path;
```

Comment:
@arvidn:  Something I wondered while looking at Libtorrent's existing UNC path implementation: 

Why are there 4 backslashes in the prefix here, rather than 2?

Reply:
in C and C++, `\` is an escape character, for `\n` for instance. To represent one backslash, you have to escape it `\\`.